### PR TITLE
sql: update transaction behavior

### PR DESCRIPTION
**Pull request by lilin90**
@tiancaiamao @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/675

### CODE REVIEW

File: sql/transaction-isolation.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ TiDB offers two transaction isolation levels: Read Committed and Repeatable Read
...
 
-Use the following command to set the level of transaction isolation:
+Use the following command to set the isolation level of the Session or Global transaction:
 
 ```
-SET SESSION TRANSACTION ISOLATION LEVEL [read committed|repeatable read]
+SET [SESSION | GLOBAL] TRANSACTION ISOLATION LEVEL [read committed|repeatable read]
+```
+
+If you do not use the Session or Global keyword, this statement takes effect only for the next executed transaction, but not for the entire session or global transaction.
```
Comment (CaitinChen) [inline sql/transaction-isolation.md]: the next executed transaction -> the transaction to be executed next

File: sql/transaction-isolation.md [hunk 2]
```diff
@@ -73,3 +79,27 @@ You can control the number of retries by configuring the `retry-limit` parameter
 # The maximum number of retries when commit a transaction.
 retry-limit = 10
 ```
+
+## Statement rollback
+
+If you execute a statement within a statement, the executed statement does not take effect when an error occurs.
```
Comment (CaitinChen) [inline sql/transaction-isolation.md]: within a statement -> within a transaction

**Comment by lilin90** [comment]
@CaitinChen PTAL

**Comment by lilin90** [comment]
@tiancaiamao PTAL

**Pull request merged**